openff/toolkit/utils/utils.py

```diff
@@ -163,10 +165,10 @@ def quantity_to_string(input_quantity: Quantity) -> str:
         The serialized quantity
 
     """
-    unitless_value: float | int | NDArray | list = input_quantity.m_as(input_quantity.units)
+    unitless_value: float | int | NDArray | list = input_quantity.m
```

Comment:
I don't think there was a reason to convert this quantity into the units it already has